gitops-deployment/README.md

```diff
@@ -9,115 +9,241 @@ For more information about ArgoCD, see the [ArgoCD documentation](https://argo-c
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+EOF
+```
+
+#### The GitOps ArgoCD instance is deployed in namespace-scoped mode:
+
+```bash
+cat <<EOF | oc apply -f -
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: gitops-websphere-automation-role
+  namespace: ${NAMESPACE}
+rules:
+  - apiGroups: ["operators.coreos.com"]
+    resources: ["operatorgroups", "subscriptions", "catalogsources"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+  - apiGroups: ["networking.k8s.io"]
+    resources: ["networkpolicies"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+  - apiGroups: ["automation.websphere.ibm.com"]
+    resources: ["websphereautomations", "webspheresecures", "webspherehealths"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+EOF
+```
+
+#### Apply the RoleBinding to link the Role to the GitOps Application Controller service account
+
+```bash
+cat <<EOF | oc create -f -
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: gitops-websphere-automation-rolebinding
+  namespace: ${NAMESPACE}
+subjects:
+- kind: ServiceAccount
+  name: ${GITOPS_SERVICEACCOUNT}
+  namespace: ${GITOPS_NAMESPACE}
+roleRef:
+  kind: Role
+  name: gitops-websphere-automation-role
+EOF
+```  
+
+### Deploy Argo CD Applications
+
+Argo CD Applications are registered with the Argo CD server to define the desired state of Kubernetes resources. Each application specifies the source repository containing the manifests and the target cluster where those resources should be deployed. To deploy IBM WebSphere Automation, the Argo CD server synchronizes the corresponding applications to the target cluster, ensuring that the defined resources are applied and maintained in the desired state.
+
+This section describes the Argo CD Applications that will be synchronized in the following steps to complete the installation. The Helm Chart Templates for IBM WebSphere Automation are hosted in a GitHub repository at https://github.com/IBM/ibm-websphere-automation/tree/main/gitops-deployment, which includes dedicated branches and release artifacts for each version. If customization is needed, you may use a forked repository as the SOURCE_REPOSITORY. Refer to the [Customized Installation](#customized-installation) section for guidance on modifying the Helm Chart Templates.
+
+- [Argo CD Custom Health Checks](#argo-cd-custom-health-checks)
+- [IBM Licensing Service](#ibm-licensing-service)
+- [IBM Cert Manager](#ibm-cert-manager)
+- [IBM WebSphere Automation](#ibm-cloud-pak-for-aiops)
+
+When configuring Argo CD applications:
+
+- Set the SOURCE_REPOSITORY to the GitHub repository containing the Helm Chart Templates.
+- Set the TARGET_REVISION to the branch name that corresponds to the desired IBM WebSphere Automation version (e.g., use 1.8.2 for WebSphere Automation v1.8.2).
```

Comment:
Is this the branch or the tag?  If the tag, we should prefix tags with `v`, e.g. (`v1.8.2`)